src/Azure/composer.json

```diff
@@ -0,0 +1,55 @@
+{
+    "name": "open-telemetry/contrib-azure",
+    "description": "The Azure package for opentelemetry-php",
+    "keywords": ["opentelemetry", "otel", "azure", "tracing", "detector", "propagator"],
+    "type": "library",
+    "license": "Apache-2.0",
+    "authors": [
+        {
+            "name": "opentelemetry-php-contrib contributors",
+            "homepage": "https://github.com/open-telemetry/opentelemetry-php-contrib/graphs/contributors"
+        }
+    ],
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "php": "^7.4 || ^8.0",
+        "guzzlehttp/guzzle": "^7.8",
```

Comment:
use psr-18 discovery (but can use guzzle as a dev dependency if it makes testing easier)